pkg/runner/tengo.go

```diff
@@ -0,0 +1,463 @@
+package runner
+
+import (
+	"os"
+	"strings"
+	"time"
+
+	tengopkg "github.com/d5/tengo/v2"
+	"github.com/d5/tengo/v2/parser"
+	"github.com/d5/tengo/v2/stdlib"
+	"github.com/mdouchement/ldt/pkg/tengolib"
+	"github.com/mdouchement/logger"
+	"github.com/pkg/errors"
+)
+
+type tengo struct {
+	base
+
+	src      string
+	redirect *os.File
+}
+
+func (r *tengo) Run() {
+	r.Lock()
+	defer r.Unlock()
+
+	// Reset previous state
+	r.err = nil
+
+	//
+
+	start := time.Now()
+	logger := r.log.WithPrefixf("[%s]", r.ctx.Name()).WithField("id", GenerateID())
+	logger.Info("Running tengo script")
+
+	// Load modules
+	modules := stdlib.GetModuleMap(stdlib.AllModuleNames()...)
+	tengolib.MergeModule(modules, tengolib.AllModuleNames()...)
+	modules.AddBuiltinModule("logger", withlogger(logger).Value)
```

Comment:
Make it global package under shigoto namespace (e.g. `shigoto.logger.Info(...)` because it's the shigoto's logger instance